packages/deployment/scripts/test-docker-build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+
+nowInMs() {
+  NO_COLOR=1 node -e "console.log(Date.now())"
+}
+
```

Comment:
Bash has this built in: https://stackoverflow.com/a/67044674/46040 Please use that.
```suggestion
```